### PR TITLE
Awesome books: with ES6

### PR DESCRIPTION
**Pull request by fasaldana**
In this milestone, the awesome book app was re-created using ES6 syntax.
- Created internal modules with a JS file.
- Used export and import.
- Added a date to the project using Luxon.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "type": "git",
+    "url": "git+https://github.com/fasaldana/awesome_books.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/fasaldana/awesome_books/issues"
+  },
+  "homepage": "https://github.com/fasaldana/awesome_books#readme"
```
Comment (GzimAsani) [inline package.json:20]: - [ ] Please add a descriptive readme file. You can use this  [template](https://github.com/microverseinc/readme-template/blob/master/README.md)

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @fasaldana        
Good job so far! You are almost done :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Good job with the gitflow :+1:
- Good PR description :clap: 

## Required Changes ♻️ 
- [ ] _Check the comments below :arrow_double_down: _

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,35 @@
...
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
```
Comment (gemmen29) [inline README.md:13]: [OPTIONAL] Kindly remove this section as there is no live demo, you add this only if there exists a live demo for the project

**Review by gemmen29 — APPROVED** [review]
Hi @fasaldana, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions:
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**